.github/workflows/release.yaml

```
@@ -0,0 +1,28 @@
+name: release
+
+on:
+  push:
+    tags:        
+      - '*' 
```

nit: remove trailing spaces 
```suggestion
    tags:
      - '*'
```